services/self-service/src/main/java/com/epam/dlab/backendapi/service/impl/UserGroupServiceImpl.java

```diff
@@ -83,6 +88,12 @@ public void removeGroupFromRole(Set<String> groups, Set<String> roleIds) {
 
 	@Override
 	public void removeGroup(String groupId) {
+		if (projectDAO.getProjectsWithStatusNotIn(ProjectDTO.Status.DELETED, ProjectDTO.Status.DELETING).stream()
+				.anyMatch(p -> p.getGroups().contains(groupId) && p.getGroups().size() == 1)) {
+			throw new ResourceConflictException("Group can not be removed because it is used in some project");
+		}
+
+		projectDAO.removeGroupInProjects(groupId);
```

Comment:
such method call not needed because admin should firstly remove group from project manually